lib/audit.js

```diff
@@ -48,10 +49,15 @@ class Audit extends ArboristWorkspaceCmd {
   }
 
   exec (args, cb) {
-    this.audit(args).then(() => cb()).catch(cb)
+    const auditType = this.npm.config.get('audit-type')
+    if (auditType === 'license') {
+      this.auditLicenses(args).then(() => cb()).catch(cb)
+    } else { 
+      this.auditAdvisories(args).then(() => cb()).catch(cb)
+    }
```

Comment:
does this mean that `npm audit` can only do _either_ licenses or advisories, but not both?

Should `audit-type` be validated somehow so it's only "license" or "advisory"?

Reply:
@ljharb in pairing with @ruyadorno today, we intentionally kept this limited to "getting it working" and not fully implementing. This was the minimum way to get it working. There will be individual ways to do advisories, licenses, and both (presuming both will be the default, since `license` is going to be opt-in rather than opt-out).

Ruy's recommendation was `npm audit --audit-type=license` as a more npm-y API, so we went with that. I'm not opposed to other APIs though and presume we'll end up bikeshedding a bit.

Reply:
When I just want one type, that totally makes sense to me - but when I want both, I'd want some config way to per-command override the global `audit-type=license` i plan to configure.

As for the default, I'd assume licenses aren't validated unless there's a config for it, which means it'd be off initially?

Reply:
oop, sorry thought I replied to this last night:

the intent is to have `npm audit` do **both** licenses and advisories by default. It can **optionally** do _just_ licenses _or_ advisories.

the implementation here is an intentionally basic one that's the result of @ruyadorno and myself pairing trying to get a _very basic_ implementation working that can be expanded on to include everything that's been scoped out, hence WIP status of the PR.

Reply:
OK - so what `audit-type` value would I provide to override a global config setting that restricts it to "just licenses"?

Reply:
`advisories`: https://github.com/npm/cli/pull/3452/files#diff-2a8ed1f0d31e4c759861aa941a2dc7e4965c10d96a79bad6499301a9a0dadbc2R56